database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -1,49 +1,48 @@
 <?php
 
-use Illuminate\Database\Migrations\Migration;
-use Illuminate\Database\Schema\Blueprint;
-use Illuminate\Support\Facades\Schema;
+namespace App\Models;
 
-return new class extends Migration
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Carbon\Carbon;
+
+class User extends Model
 {
-    /**
-     * Run the migrations.
-     */
-    public function up(): void
+    use HasFactory;
+
+    protected $table = 'users';
```

Comment:
kayak gini contohnya:
    protected string $table;

    public function __construct()
    {
        $this->table = config('db_tables.financial_account');
    }